apps/docs/content/components/Hero.mdx

```diff
@@ -18,53 +27,73 @@ import {Hero} from '@primer/react-brand'
 ### Default
 
 ```jsx live
-<Hero
-  heading="This is my super sweet hero heading"
-  description="Lorem ipsum dolor sit amet, consectetur adipiscing elit. In sapien sit ullamcorper id. Aliquam luctus sed turpis felis nam pulvinar risus elementum."
-  primaryAction={{
-    text: 'Primary action',
-    href: '#',
-  }}
-  secondaryAction={{
-    text: 'Secondary action',
-    href: '#',
-  }}
-  align="center"
-/>
+<Hero>
+  <Hero.Heading>This is my super sweet hero heading</Hero.Heading>
+  <Hero.Description>
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit. In sapien sit
+    ullamcorper id. Aliquam luctus sed turpis felis nam pulvinar risus
+    elementum.
+  </Hero.Description>
+  <Hero.PrimaryAction href="#">Primary action</Hero.PrimaryAction>
+  <Hero.SecondaryAction href="#">Secondary action</Hero.SecondaryAction>
+</Hero>
 ```
 
 ### Alignment
 
 `Hero` text alignment can be adjusted by changing the `align` prop.
 
 ```jsx live
-<Hero
-  heading={
+<Hero>
```

Comment:
This example seems to be the same as the default one. Could we please update to use center alignment?

```suggestion
<Hero align="center">
```